khal/khalendar/event.py

```diff
@@ -599,6 +599,11 @@ def format(self, format_string, relative_to, env={}, colors=True):
             for c in ["black", "red", "green", "yellow", "blue", "magenta", "cyan", "white"]:
                 attributes[c] = attributes[c + '-bold'] = ''
 
+        if "control codes":
```

Comment:
What is this if statement doing? "control codes" will always be True.

Reply:
This was done to keep the code at the same tab level as the other `attributes` modifications, as well as to allow you to extend this to be toggle-able in the future (ie: allow some users to turn it off for whatever reason).

I'd be happy remove the logic if you'd prefer, no problem.